src/main/resources/schemas/master_schema.json

```diff
@@ -2006,117 +868,43 @@
                                       },
                                       "offence": {
                                         "title": "offence",
-                                        "description": "The offence and details of offences associated with the case. It is also known as charges.",
                                         "type": "array",
                                         "items":{
                                           "type": "object",
                                           "properties": {
-                                            "offenceCode":{
-                                              "description": "Offence unique code",
-                                              "title": "Offence code",
-                                              "type": "string",
-                                              "default": "",
-                                              "examples": [
-                                                "dd01-01"
-                                              ],
-                                              "pattern": "^(?!.*<[^>]+>).*$"
-                                            },
                                             "offenceTitle":{
-                                              "description": "Short description of offence",
                                               "title": "Offence title",
                                               "type": "string",
-                                              "default": "",
-                                              "examples": [
-                                                "drink driving"
-                                              ],
                                               "pattern": "^(?!.*<[^>]+>).*$"
                                             },
                                             "offenceWording":{
-                                              "description": "Long description of offence",
                                               "title": "Offence wording",
                                               "type":"string",
-                                              "default": "",
-                                              "examples": [
-                                                "driving whilst under the influence of alcohol"
-                                              ],
-                                              "pattern": "^(?!.*<[^>]+>).*$"
-                                            },
-                                            "offenceMaxPen":{
```

Comment:
Same as the previous comment. This is in still in validation schema for Magistrates standard list and test files